test/valid_field_test.exs

```diff
@@ -24,7 +24,7 @@ defmodule ValidFieldTest do
       |> ValidField.assert_invalid_field(:title, ["", nil, "Something else"])
     end
 
-    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<2016-08-23>", fn ->
+    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<#{get_current_date}>", fn ->
```

Comment:
@BlakeWilliams 👋 
I was looking at https://github.com/DockYard/valid_field/pull/21/files and wasn't sure why I had to make this change to get the tests to pass.